src/lib/getDownloads.ts

```diff
@@ -106,4 +106,24 @@ export const sendMoveVideoQueuedIgnored = async (token: string, videoId: string,
     throw new Error("Error moving video to" + status + ".");
   }
   return response.json();
+};
+
+export const sendTasks = async (token: string, task: string): Promise<Task> => {
```

Comment:
Does the `task: string` param have a set list of `tasks` that it can take? We can turn that into a type if you wanna be more strict about it and make them easier to use in the IDE.

```ts
type Tasks = "task1" | "task2" | "task3"
```

Reply:
Good idea, I forgot you can do that.